modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalCardTag.java

```diff
@@ -0,0 +1,821 @@
+
+	public String getSubtitle() {
+		if ((_subtitle == null) && (_verticalCard != null)) {
+			return _verticalCard.getSubtitle();
+		}
+
+		return _subtitle;
+	}
+
+	public String getTitle() {
+		if ((_title == null) && (_verticalCard != null)) {
+			return _verticalCard.getTitle();
+		}
+
+		return _title;
+	}
+
+	public VerticalCard getVerticalCard() {
+		return _verticalCard;
+	}
+
+	public void setActionDropdownItems(List<DropdownItem> actionDropdownItems) {
+		_actionDropdownItems = actionDropdownItems;
+	}
+
+	public void setDisabled(Boolean disabled) {
+		_disabled = disabled;
+	}
+
+	public void setFlushHorizontal(boolean flushHorizontal) {
+		_flushHorizontal = flushHorizontal;
+	}
+
+	public void setFlushVertical(boolean flushVertical) {
+		_flushVertical = flushVertical;
+	}
+
+	/**
+	 * @deprecated As of Cavanaugh (7.4.x), with no direct replacement
+	 */
+	@Deprecated
+	public void setGroupName(String groupName) {
+		_groupName = groupName;
+	}
+
+	public void setHref(String href) {
+		_href = href;
+	}
+
+	public void setIcon(String icon) {
+		_icon = icon;
+	}
+
+	public void setImageAlt(String imageAlt) {
+		_imageAlt = imageAlt;
+	}
+
+	public void setImageSrc(String imageSrc) {
+		_imageSrc = imageSrc;
+	}
+
+	public void setInputName(String inputName) {
+		_inputName = inputName;
+	}
+
+	public void setInputValue(String inputValue) {
+		_inputValue = inputValue;
+	}
+
+	public void setLabels(List<LabelItem> labels) {
+		_labels = labels;
+	}
+
+	public void setLabelStylesMap(Map<String, String> labelStylesMap) {
+		_labelStylesMap = labelStylesMap;
+	}
+
+	public void setSelectable(Boolean selectable) {
+		_selectable = selectable;
+	}
+
+	public void setSelected(Boolean selected) {
+		_selected = selected;
+	}
+
+	/**
+	 * @deprecated As of Cavanaugh (7.4.x), with no direct replacement
+	 */
+	@Deprecated
+	public void setSpritemap(String spritemap) {
+		_spritemap = spritemap;
+	}
+
+	public void setStickerCssClass(String stickerCssClass) {
+		_stickerCssClass = stickerCssClass;
+	}
+
+	public void setStickerIcon(String stickerIcon) {
+		_stickerIcon = stickerIcon;
+	}
+
+	public void setStickerImageAlt(String stickerImageAlt) {
+		_stickerImageAlt = stickerImageAlt;
+	}
+
+	public void setStickerImageSrc(String stickerImageSrc) {
+		_stickerImageSrc = stickerImageSrc;
+	}
+
+	public void setStickerLabel(String stickerLabel) {
+		_stickerLabel = stickerLabel;
+	}
+
+	public void setStickerShape(String stickerShape) {
+		_stickerShape = stickerShape;
+	}
+
+	public void setStickerStyle(String stickerStyle) {
+		_stickerStyle = stickerStyle;
+	}
+
+	public void setSubtitle(String subtitle) {
+		_subtitle = subtitle;
+	}
+
+	public void setTitle(String title) {
+		_title = title;
+	}
+
+	public void setVerticalCard(VerticalCard verticalCard) {
+		_verticalCard = verticalCard;
+	}
+
+	@Override
+	protected void cleanUp() {
+		super.cleanUp();
+
+		_actionDropdownItems = null;
+		_disabled = false;
+		_flushHorizontal = false;
```

Comment:
Thanks. 
Just another question: In the `cleanup` method and in the initial variable declaration ([at the very end of the file](https://github.com/liferay-frontend/liferay-portal/blob/c6d092ef1b30a32b392a92b01cfbf732b903affa/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalCardTag.java#L794-L819)) should all variables should have the value of `null` or no value at all?

 
I'm asking because if I only change their value in the `cleanup` method, when running `gradlew formatSource` I get a warning like this 

```sh
 Initial value for '_flushHorizontal' differs from value in cleanUp method, see https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/src/main/resources/documentation/checks/java_clean_up_method_variables_check.markdown
```

Which is explained [here](https://github.com/liferay/liferay-portal/blob/9f782bf5a3a61a1374b180a1a6050aff35917261/modules/util/source-formatter/src/main/resources/documentation/checks/java_clean_up_method_variables_check.markdown)

Reply:
Just to leave this somewhere, @carloslancha helped me figure it out.

Since we're using "model" classes these variable should have no value when declared and be reset to `null` in the `cleanup` method. Without model classes, the default value would be used when declaring the variable and in `cleanup` we would reset them to their original value.